frc53_nft/src/types.rs

```diff
@@ -97,13 +104,14 @@ impl RecipientData for MintIntermediate {
     }
 }
 
-/// Intermediate data used by transfer_return to construct the return data
+/// Intermediate data used by [`NFT::transfer_return`][`super::NFT::transfer_return`] to construct
```

Comment:
what's going on with both of these links? is this a typo that they're both included?

Reply:
It's markdown formatting (reference style link). This is the explicit linking I was avoiding in https://github.com/filecoin-project/actors-utils/pull/237#discussion_r1704844288.

Rustdoc has some interesting handling of links: https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html#linking-to-items-by-name

(I guess the backticks around the reference aren't necessary, but they work)